vault.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Vault",
+    "plugin_schema": "2",
+    "release": "11.3-RELEASE",
```

Comment:
Should we maybe use `12.1-RELEASE` ?

Reply:
Gah, I missed that earlier. Thanks, will let the submitter know.